ToDoApp/docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: "3"
+
+services:
+  postgres:
+    image: postgres
+    container_name: postgres
+    volumes:
+      - .data/:/var/lib/postgresql/
+    restart: always
+    ports:
+      - "5432:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${DB_USERNAME} -p $DB_PORT -d ${DB_NAME}"]
+      interval: 15s
+      timeout: 5s
+      retries: 5
+    environment:
+      - POSTGRES_USER=${DB_USERNAME}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_DB=${DB_NAME}
+    networks:
+      - frcp
+    env_file:
+      - .env
+
+  rabbitmq:
+    container_name: rabbitmq
+    hostname: rabbitmq
+    image: rabbitmq:management-alpine
+    restart: always
+    healthcheck:
+      test: rabbitmq-diagnostics -q ping
+      interval: 15s
+      timeout: 5s
+      retries: 5
+    environment:
+      - RABBITMQ_DEFAULT_USER=${RABBITMQ_USER}
+      - RABBITMQ_DEFAULT_PASS=${RABBITMQ_PASSWORD}
+      - RABBITMQ_DEFAULT_VHOST=/
+    networks:
+      - frcp
+    env_file:
+      - .env
+
+  worker:
+    container_name: worker
+    build: ./app
+    command: celery -A main.celery worker --loglevel=info
+    depends_on:
+      rabbitmq:
+          condition: service_healthy
+      postgres:
+        condition: service_healthy
+    environment:
+      - CELERY_BROKER_URL=${CELERY_BROKER_URL}
+      - CELERY_RESULT_BACKEND=${CELERY_RESULT_BACKEND}
+    volumes:
+      - ./app:/app
+    networks:
+      - frcp
+    env_file:
+      - .env
+
+  fastapi:
+    container_name: fastapi
+    build: ./app
+    volumes:
+      - ./app:/app
+    ports:
+      - "8080:8000"
```

Comment:
ports should be coming from .env and same port should be using when starting api server through uvicorn